std/encoding/base64url.ts

```diff
@@ -1,46 +1,47 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
-import {
-  decode as convertBase64ToArrayBuffer,
-  encode as convertArrayBufferToBase64,
-} from "./base64.ts";
+import * as base64 from "./base64.ts";
 
 /*
  * Some variants allow or require omitting the padding '=' signs:
  * https://en.wikipedia.org/wiki/Base64#URL_applications
+ * @param string
  */
-export function addPaddingToBase64url(base64url: string): string {
-  if (base64url.length % 4 === 2) return base64url + "==";
-  if (base64url.length % 4 === 3) return base64url + "=";
-  if (base64url.length % 4 === 1) {
-    throw new TypeError("Illegal base64url string!");
+export function addPaddingToBase64url(b64url: string): string {
+  if (b64url.length % 4 === 2) return b64url + "==";
+  if (b64url.length % 4 === 3) return b64url + "=";
+  if (b64url.length % 4 === 1) {
+    throw new TypeError("Illegal b64url string!");
   }
-  return base64url;
+  return b64url;
```

Comment:
Seems like not needed change that makes diff bigger

Reply:
My reason was uniformity compared to the base32 module. But point taken. Do you still want me to change this?

Reply:
Yes, please